generated/nifake/nifake/_converters.py

```diff
@@ -280,3 +281,27 @@ def convert_double_each_element(numbers):
     return [x * 2 for x in numbers]
 
 
+def convert_from_enum_with_converter_enum(value):
+    try:
+        return {
+            enums.EnumWithConverter.RED: True,
+            enums.EnumWithConverter.BLUE: False,
+            enums.EnumWithConverter.YELLOW: 'yellow',
+            enums.EnumWithConverter.BLACK: 42,
+        }[value]
+    except Exception:
+        raise ValueError(f"EnumWithConverter enum value '{value}' cannot be converted")
+
+
+def convert_to_enum_with_converter_enum(value):
+    try:
+        return {
+            True: enums.EnumWithConverter.RED,
+            False: enums.EnumWithConverter.BLUE,
+            'yellow': enums.EnumWithConverter.YELLOW,
+            42: enums.EnumWithConverter.BLACK,
+        }[value]
+    except Exception:
```

Comment:
shouldn't this be key error?

Reply:
I have updated this to KeyError in my latest commit